site/content/integrate/slash-commands/_index.md

```diff
@@ -88,6 +88,8 @@ Slash command responses support more than just the `text` field. Here is a full
 | type | Sets the post `type`, mainly for use by plugins.<br> If not blank, must begin with `custom_`. Passing `attachments` will ignore this field and set the type to `slack_attachment`. | No |
 | extra\_responses | An array of responses used to send more than one post in your response. Each item in this array takes the shape of its own command response, so it can include any of the other parameters listed here, except `goto\_location` and `extra\_responses` itself. Available in Mattermost v5.6 and later. | No |
 | props | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br> Mainly used by other integrations accessing posts through the REST API.<br> The following keys are reserved: `from_webhook`, `override_username`, `override_icon_url` and `attachments`. | No |
+| skip_slack_parsing | Signals Mattermost that the post is correctly escaped for Mattermost and the Slack-compatibility logic should be skipped. Useful if the post contains code which is incorrectly handled by the Slack-compatibility logic. | No |
```

Comment:
Would you please move this in front of `props`?

Reply:
ACK. For future reference: what is the a reason for it? How are the entries ordered?

Reply:
`Props` is like the "appendix". All information that doesn't belong in a specific field gets added there. This is why it should be last.